ChangeLog

```diff
@@ -1,3 +1,16 @@
+Wed Jan 17 23:55:29 UTC 2018  Frederick Hornsey <fred@hornsey.us>
+
+        * config/wireshark.mpb:
+        * config/global.features:
+
+		  When using Wireshark builds built using Autoconf (like Linux),
```

Comment:
tabs -> spaces here

Reply:
That's strange since I wrote this in vim set to 4 spaces for tabs.